### PR TITLE
Setup React App

### LINKED ISSUE #10: Frontend [2pt] Setup React app
- Install react app
- Gitflow
- Linters for React
- Install Redux and Redux toolkit.

### PR DESCRIPTION
**Pull request by ernestmusong**
In this PR, I've done the following:

- Created a react app
- Installed common dependencies
- Installed Redux
- created common folders

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+PORT = 8000
```
Comment (PowerLevel9000) [inline .env]: - [ ] Kindly make sure the env file should be in `gitignore` file, as it may contain sensitive information of the project 😉😉💪

**Review by PowerLevel9000 — CHANGES_REQUESTED** [review]
# 🟡🟡🟡 Required Changes ♻️
# Hi @ernestmusong my dear team member, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereGIF

## Highlights :heavy_check_mark: 
- [x] Git flow is used correctly :white_check_mark: 
- [x] Correct setup of the project expected with skelton code :white_check_mark: 
- [x] Descriptive pull request title and summary :white_check_mark:   

## Required Changes ♻️
Also, Check the comments under the review._

- [ ] I can see some of the comment codes and file which comes with CRA and that won't be part of our project make sure to clean them please 💕💕😁

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: .env (hunk 1 above)
Comment (ernestmusong) [reply .env]: Thanks @PowerLevel9000 for the catch. I have now added the file

**Review by PowerLevel9000 — APPROVED** [review]
# 🟢🟢🟢Status Approved  🟢🟢🟢
# Hi @ernestmusong  👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


AgreeSealGIF
## Highlights
- No Linters error ✅
- Followed GitHub flow ✅
- Great working on the Changes Requiered✅
- Descriptive PR ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**